ui/src/app/app.module.ts

```diff
@@ -27,7 +28,13 @@ import { NgSelectModule } from '@ng-select/ng-select';
     NgbModule,
     HttpClientModule,
     FontAwesomeModule,
-    NgSelectModule
+    NgSelectModule,
+    ServiceWorkerModule.register('custom-service-worker.js', {
```

Comment:
How does adding a service worker affect the regular scenario of loading the application in a web browser? Is there anything different that the users are seeing (popups asking for notification permissions or anything like that)?